docs/UserGuide.md

```diff
@@ -14,18 +14,17 @@ Amigos is a desktop application to help tech-savvy university students manage th
 1. Ensure you have Java `11` or above installed in your Computer.
 
 
-1. Download the latest `addressbook.jar` from [here](https://github.com/se-edu/addressbook-level3/releases).
+1. Download the latest `amigos.jar` from [here](https://github.com/AY2122S2-CS2103-F09-2/tp/releases/tag/v1.1).
 
-1. Copy the file to the folder you want to use as the _home folder_ for your AddressBook.
+1. Copy the file to the folder you want to use as the _home folder_ for your application.
 
 1. Double-click the file to start the app. The GUI similar to the below should appear in a few seconds. Note how the app contains some sample data.<br>
    ![Ui](images/Ui.png)
 
-1. Type the command in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will open the help window.<br>
-   Some example commands you can try:
-
-   * **`list`** : Lists all contact. # to change
+1. Type the command in the command box and press Enter to execute it. e.g:
 
+   * **`showfriends`** : Lists all friends.
+   
 
 1. Refer to the [Features](#features) below for details of each command.
 
```

Comment:
In Line 40, the 'add' command refers to a command that does not exist in Amigos. Hence, would be good to change it!